gtsfm/common/two_view_estimation_report.py

```diff
@@ -46,3 +45,6 @@ class TwoViewEstimationReport:
     U_error_deg: Optional[float] = None
     i2Ri1: Optional[Rot3] = None
     i2Ui1: Optional[Unit3] = None
+    reproj_error_gt_model: Optional[np.ndarray] = None
+    inlier_avg_reproj_error_gt_model: Optional[float] = None
+    outlier_avg_reproj_error_gt_model: Optional[float] = None
```

Comment:
We still need to add definitions for each of these new variables to the class in the docstring.

Reply:
added